### ISSUE TITLE
Reqirements

### ISSUE THREAD

**Issue by Craftefix** [uhlin/enhanced-duc#1]
This aplication needs g++
hope that this is still in develompment

**Comment by uhlin** [comment]
Yes, it requires a C++ compiler, not necessarily g++. Would you like me to document that in the README?
Regarding the development the program does what I want currently so no new version is planned, but that can change with time... if you have a feature request file one.

**Comment by uhlin** [comment]
No response, closing...
